modules/custom/az_migration/migrations/migrate_plus.migration.az_paragraph_card_deck.yml

```diff
@@ -17,46 +17,71 @@ destination:
 process:
 
   field_az_cards:
-    -
-      plugin: sub_process
-      source: field_uaqs_photo
-      process:
-        delta: delta
-        media: 
-          - 
-            plugin: migration_lookup
-            source: fid
-            migration: 
-              - az_media
-    -
-      plugin: paragraphs_field_property_mapping
-      key: title
-      field_name: field_uaqs_short_title
-      value: value
-    -
-      plugin: paragraphs_field_property_mapping
-      key: body
-      field_name: field_uaqs_summary
-      value: value
-    -
-      plugin: paragraphs_field_property_mapping
-      key: body_format
-      default_value: az_standard
-    -
-      plugin: paragraphs_field_property_mapping
-      key: link_title
-      field_name: field_uaqs_links
-      value: title
-    -
-      plugin: paragraphs_field_property_mapping
-      key: link_uri
-      field_name: field_uaqs_links
-      value: url
-    -
-      plugin: paragraphs_field_property_mapping
-      key: options
-      default_value: 
-        class: 'bg-white'
+    source: field_uaqs_cards_values
+    plugin: sub_process
+    process:
+      delta: delta
+      title:
+        -
+          plugin: skip_on_empty
+          method: process
+          source: field_uaqs_short_title
+        -
+          plugin: extract
+          index:
+            - 0
+            - value
+      body:
+        -
+          plugin: skip_on_empty
+          method: process
+          source: field_uaqs_summary
+        -
+          plugin: extract
+          index:
+            - 0
+            - value
+      body_format:
+        plugin: default_value
+        default_value: az_standard
+      link_title:
+        -
+          plugin: skip_on_empty
+          method: process
+          source: field_uaqs_links
```

Comment:
I think we should add something to the README.md  for this migration that says we are only migrating the first link. If there are multiple links on a card, they can edit the migrated card after the migration, to add the links to the text area as HTML instead of using the link field.